blog/well-known-change-password.md

```diff
@@ -0,0 +1,23 @@
+# Build a discoverable change password form
+
+We can use a well-known URL in our websites to make our change password forms discoverable by tools, like 1Password, LastPass, Bitwarden, etc.
```

Comment:
Maybe change password could use italics or be written as change-password

Reply:
```suggestion
We can use a well-known URL in our websites to make our _change password_ forms discoverable by tools, like 1Password, LastPass, Bitwarden, etc.
```